lib/nitcorn/http_request.nit

```diff
@@ -82,7 +82,7 @@ class HttpRequest
 	fun int_arg(arg_name: String): nullable Int do
 		if not all_args.has_key(arg_name) then return null
 		var i = all_args[arg_name]
-		if not i.is_numeric then return null
+		if not i.is_int then return null
 		return i.to_i
```

Comment:
??? The doc says it clearly that the method accepts any number, and converting floats to integers is not surprising.

@jcbrinfo, just so you know, even if we have nullables in Nit, when designing API we avoid returning nulls as much as possible. For instance, if the client wants an integer and the value can be an integer, we return an integer! That's POLA.